additions.c

```diff
@@ -5,6 +5,36 @@
 #include <GLUT/glut.h>
 #include "quickjs/quickjs.h"
 
+#ifdef WEBGL_DEBUG
+#include <stdio.h>
+
+static void debug_check_error(JSContext *ctx, const char *func, int argc, JSValueConst *argv) {
+    GLenum err = glGetError();
+    if (err != GL_NO_ERROR) {
```

Comment:
```suggestion
    if (err != GL_NO_ERROR || WEBGL_DEBUG > 1) {
```

Let's make an improvement: allow to specify `-DWEBGL_DEBUG=2` for compiler options and get unconditional OpenGL calls log mode.